crates/trigger-http/src/lib.rs

```diff
@@ -1000,3 +1007,212 @@ mod tests {
         assert!(req.headers().get("Host").is_some());
     }
 }
+
+/// Translate a [`hyper::Error`] to a wasi-http `ErrorCode` in the context of a request.
+pub fn hyper_request_error(err: hyper::Error) -> ErrorCode {
```

Comment:
nit: can you move `hyper_request_error` and `dns_error` down to the bottom of the file so that `default_send_request_handler` is further up in the file?

Reply:
done.